front/src/Phaser/Game/GameScene.ts

```diff
@@ -639,7 +662,11 @@ export class GameScene extends ResizableScene implements CenterListener {
 
                 const openWebsiteTriggerValue = allProps.get(TRIGGER_WEBSITE_PROPERTIES);
                 if(openWebsiteTriggerValue && openWebsiteTriggerValue === ON_ACTION_TRIGGER_BUTTON) {
-                    layoutManager.addActionButton('openWebsite', 'Click on SPACE to open the web site', () => {
+                    let message = allProps.get(WEBSITE_MESSAGE_PROPERTIES);
+                    if(message === undefined){
+                        message = 'Click on SPACE to open the web site';
```

Comment:
should be "hit" or "press". a "click" is always done with the mouse, not the keyboard